dashboard/forms.py

```diff
@@ -1,20 +1,25 @@
 from email.policy import default
 from django import forms
 from django.contrib.auth.forms import UserCreationForm
-from django.contrib.auth.models import User
+from django.contrib.auth import get_user_model  
+User = get_user_model()  
```

Comment:
Dont call function directly inside a module. call it in function

Reply:
update